app/src/lib/progress/from-process.ts

```diff
@@ -23,8 +23,13 @@ export async function executionOptionsWithProgress(
   let lfsProgressPath = null
   let env = {}
   if (options.trackLFSProgress) {
-    lfsProgressPath = await createLFSProgressFile()
-    env = { GIT_LFS_PROGRESS: lfsProgressPath }
+    try {
+      lfsProgressPath = await createLFSProgressFile()
+      env = { GIT_LFS_PROGRESS: lfsProgressPath }
+    } catch (e) {
+      console.log('Error writen LFS log: ' + e)
```

Comment:
Perhaps use the `log` global? `log.error('Error writing LFS progress file:', e)`

Reply:
Example from another part of the codebase:
https://github.com/desktop/desktop/blob/614f8489850fd1f409a41490c8a983f8c8160017/app/src/lib/stats/stats-store.ts#L143-L145